psalm-baseline.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<files psalm-version="5.7.7@e028ba46ba0d7f9a78bc3201c251e137383e145f">
+  <file src="test/StoragelessTest/Session/LazySessionTest.php">
+    <InvalidFunctionCall>
+      <code><![CDATA[($this->sessionLoader)()]]></code>
```

Comment:
So, when I move from:
```php
    /** @var MakeSession&MockObject */
    private MakeSession $sessionLoader;
```
to
```php
    /** @var MakeSession&MockObject */
    private MakeSession&MockObject $sessionLoader;
```
Tests of course pass, and PSalm doesn't need the baseline.

But then `phpcbf` removed the DocBlock, and for some unbeknownst reason, PSalm breaks :cry:

Reply:
I'd prioritize psalm over phocbf here: we can tell phpcbf to shut up here, and remove the baseline instead.

Rationale: this project should remain baseline -less.